src/packages/shared-types/actions.ts

```diff
@@ -10,6 +10,8 @@ export enum Action {
   WITHDRAW_PACKAGE = "withdraw-package",
   REMOVE_APPK_CHILD = "remove-appk-child",
   TEMP_EXTENSION = "temporary-extension",
+  LEGACY_ADMIN_CHANGE = "legacy-admin-change",
+  LEGACY_WITHDRAW_RAI_REQUEST = "legacy-withdraw-rai-request"
```

Comment:
Not sure how to reconcile, but this enum is used primarily to represent actions taken thru the action menus, and this is the first break of that pattern. It feels like these are slightly out of place here.